src/pages/inward-transfer/forms/InwardTransferForm.js

```diff
@@ -0,0 +1,1093 @@
+import { Checkbox, FormControlLabel, Grid } from '@mui/material'
+import { useContext, useEffect, useState } from 'react'
+import toast from 'react-hot-toast'
+import CustomDatePicker from 'src/components/Inputs/CustomDatePicker'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+import CustomTextArea from 'src/components/Inputs/CustomTextArea'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import FieldSet from 'src/components/Shared/FieldSet'
+import FormShell from 'src/components/Shared/FormShell'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { useError } from 'src/error'
+import { useWindow } from 'src/windows'
+import { useDocumentType } from 'src/hooks/documentReferenceBehaviors'
+import { useForm } from 'src/hooks/form'
+import { useInvalidate } from 'src/hooks/resource'
+import { formatDateFromApi, formatDateToApi } from 'src/lib/date-helper'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { RemittanceOutwardsRepository } from 'src/repositories/RemittanceOutwardsRepository'
+import { CurrencyTradingSettingsRepository } from 'src/repositories/CurrencyTradingSettingsRepository'
+import { RemittanceSettingsRepository } from 'src/repositories/RemittanceRepository'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { DataSets } from 'src/resources/DataSets'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { SystemFunction } from 'src/resources/SystemFunction'
+import * as yup from 'yup'
+import { ControlContext } from 'src/providers/ControlContext'
+import CloseForm from './CloseForm'
+
+export default function InwardTransferForm({ labels, recordId, access, plantId, window, userId, dtId }) {
+  const [editMode, setEditMode] = useState(!!recordId)
```

Comment:
replace editMode and isClosed states into const
example: 
const isClosed= formik.values.wip === 2